src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1108,7 +1109,7 @@ def set_up_service_bindings(self):
         if self.get_argument_service_bindings() is not None:
             linker_client = get_linker_client(self.cmd)
 
-            service_connectors_def_list, service_bindings_def_list = parse_service_bindings(self.cmd, self.get_argument_service_bindings(), self.get_argument_resource_group_name(), self.get_argument_name(), self.get_argument_customized_keys())
+            service_connectors_def_list, service_bindings_def_list = parse_service_bindings(self.cmd, self.get_argument_service_bindings(), self.get_argument_resource_group_name(), self.get_argument_name(), self.containerapp_def, self.get_argument_customized_keys())
```

Comment:
Why need to change here?

Reply:
As we dicussed in another conversation, we need the `environment_id` to list all Java components, let's discuss the details in another conversation, will close this one.